pep-0660.rst

```diff
@@ -157,11 +158,19 @@ install. This section provides examples and is not normative.
   a path importable, often including the project's own ``setup.py`` and other
   scripts that would not be part of a normal installation. The proxy strategy
   can achieve a higher level of fidelity than path-based methods.
+* Symbolic links are another useful mechanism to realize editable installs.
+  Since, at the time this writing, the ``wheel`` specification does not support
+  symbolic links, they are not directly usable. It is however possible for the
+  backend to create an symlink structure in a build directory of the source
+  tree such as ``.editable/$COMPATIBILITY_TAGS``, and add that directory to the
+  python path via a ``.pth`` file in the "editable" wheel. Alternatively it is
+  also possible to defer the creation of symbolic links to an import hook or
+  executable ``.pth`` that the backend adds in the "editable" wheel.
```

Comment:
The symbolic links section is not very inspiring. We could own our lack of support for symlinks or finally add symlinks to wheel (or encourage someone to add them in the future).

I don't understand the $COMPATIBILITY_TAGS variable.

I wouldn't complain if the backend produced build/editable/ with one or two links to the top-level packages we export; add that to .pth. Solving the 'accidentally import setup.py' problem. I would be confused if a complete tree of links to all our .py was in there.

The deferred idea is best removed. It's possible but weird.